bin/rails-sandbox

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env ruby
 
-app_root = 'sandbox'
+app_root = ENV['GENERATE_FRONTEND'] ? 'sandbox-generated' : 'sandbox'
```

Comment:
@waiting-for-dev Regarding https://github.com/nebulab/solidus_starter_frontend/pull/171#discussion_r684947097, I was able to simplify the `bin/rails` script and allow it to create the sandbox-generated app and dispatch to it immediately. Here is the usage for the `bin/rails` script:

* `bin/rails g` - Dispatches to gem Rails engine
* `bin/rails ARGV` - Creates sandbox if needed and dispatches ARGV to it.
* `GENERATE_FRONTEND=1 bin/rails ARGV` - Creates sandbox-generated if needed and dispatches ARGV to it.

Reply:
Amazing, @gsmendoza !! :clap: :clap: :clap: